### PR TITLE
Portfolio Application Accessibility

### PR DESCRIPTION
**Pull request by Khawanin**
## Project Requirements ♻️ 
Check all of the items listed in the article [https://www.w3.org/WAI/test-evaluate/preliminary/](url) to check for application accessibility. 😃 
### In this project we checked for: ⬇️ 

1. Page Title - Done ✔️ 
2. Image text alternatives - Done ✔️ 
3. Text Headings - No issues found ✔️ 
4. Color Contrast - No issues found ✔️ 
5. Resize - No issues found ✔️ 
6. Moving Content - No issues found ✔️ 
7. The basic structure of the page  - No issues found ✔️ 

**We did the above activities using pair programming @Khawanin and @ShinhyoBelok**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,188 +1,202 @@
...
-                </ul>
-                <button class="button">See Project</button>
+                    <button class="button">See Project</button>
             </section>
-            <!-- projecto 3 -->
+            <!-- project 3 -->
             <section class="project">
-                <img class="snap" src="img\Snapshoot Portfolio 3.svg" alt="Project picture">
+                <img class="snap" src="img\Snapshoot Portfolio 3.svg" alt="Third project card picture">
                 <h2>Tonic</h1>
```
Comment (cynthiainga) [inline index.html]: - [ ] The page has at least one heading `<h1>, and here you use two different `hx` in one tag please kindly fix that 🙃.

File: index.html [hunk 2]
```diff
@@ -1,188 +1,202 @@
...
-                </ul>
-                <button class="button">See Project</button>
+                    <button class="button">See Project</button>
             </section>
-            <!-- projecto 2 -->
+            <!-- project 2 -->
             <section class="project">
-                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 2.svg" alt="Project picture">
+                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 2.svg" alt="Second project card picture">
                 <h2>Multi-Post Stories</h1>
```
Comment (cynthiainga) [inline index.html]: - [ ] The page has at least one heading `<h1>`, and here you use two different `hx` in one tag please kindly fix that 🙃. `eg: <h2>Multi-Post Stories</h2>` 👍 👍

File: index.html [hunk 3]
```diff
@@ -1,188 +1,202 @@
...
-                </ul>
-                <button class="button">See Project</button>
+                    <button class="button">See Project</button>
             </section>
-            <!-- projecto 4 -->
+            <!-- project 4 -->
             <section class="project">
-                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 4.svg" alt="Project picture">
+                <img class="snap reverseGrid" src="img\Snapshoot Portfolio 4.svg" alt="Fourth project card picture">
                 <h2>Multi-Post Stories</h1>
```
Comment (cynthiainga) [inline index.html]: - [ ] The page has at least one heading `<h1>`, and here you use two different `hx` in one tag please kindly fix that 🙃. `eg: <h2>Multi-Post Stories</h2>` 👍 👍

File: index.html [hunk 4]
```diff
@@ -1,188 +1,202 @@
...
-                    <li>css</li>
-                    <li>javaScript</li>
-                </ul>
-                <button class="button">See Project</button>
+                    <button class="button">See Project</button>
             </section>
         </section>
         <section class="presentation" id="aboutContainer">
             <section class="content" id="about">
                 <h1>About<br>Myself</h1>
```
Comment (cynthiainga) [inline index.html]: - [ ] The page has only one heading `<h1>` please kindly change your `h1` to another `hx` tag. `eg: <h2>About<br>Myself</h2>` 👍 👍

File: index.html [hunk 5]
```diff
@@ -1,188 +1,202 @@
...
                 <article class="dropdownHead">
                     <h4>Skills</h4>
-                    <img src="img\Unionside.svg">
+                    <img src="img\Unionside.svg" alt="Side vector icon">
                 </article>
             </section>
         </section>
     </main>
     <section class="contact">
         <h1>Contact me</h1>
```
Comment (cynthiainga) [inline index.html]: - [ ] The page has only one heading `<h1>` please kindly change your `h1` to another `hx` tag. `eg: <h2>Contact me</h2>` 👍 👍

File: index.html [hunk 6]
```diff
@@ -1,188 +1,202 @@
...
                 </ul>
             </section>
         </section>
         <section id="work">
-            <!-- projecto 1 -->
+            <!-- project 1 -->
             <section class="project">
-                <img class="snap" src="img\Snapshoot Portfolio.svg" alt="Project picture">
+                <img class="snap" src="img\Snapshoot Portfolio.svg" alt="First project card picture">
                 <h2>Tonic</h1>
```
Comment (cynthiainga) [inline index.html]: - [ ] The page has at least one heading `<h1>`, and here you use two different `hx` in one tag please kindly fix that 🙃. `eg: <h2>Tonic</h2>` 👍 👍

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Khawanin & @ShinhyoBelok, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- Change the title of the project in your `README` file, you can put for example `Portfolio` instead of `Hello Microverse` 😁 😁

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**